NodeJS/Week-1/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "week-1",
+  "version": "1.0.0",
+  "main": "avg.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "express": "^4.21.0",
+    "knex": "^3.1.0",
+    "mysql2": "^3.11.3",
+    "nodemon": "^3.1.4"
+  }
+}
```

Comment:
In this package we probably dont need `express` `knex` `mysql2`, also `nodemon` should be in the devDependencies.